.github/workflows/build.yml

```diff
@@ -80,6 +80,22 @@ jobs:
       - uses: hashicorp/setup-terraform@v2
         with:
           terraform_version: ${{ steps.setup-env.outputs.terraform-version }}
+      # Begin Go packages install below
+      - name: Install GoCritic
+        env:
+          PACKAGE_URL: github.com/go-critic/go-critic/cmd/gocritic
+          PACKAGE_VERSION: ${{ steps.setup-env.outputs.go-critic-version }}
+        run: go install ${PACKAGE_URL}@${PACKAGE_VERSION}
+      - name: Install GoSec
+        env:
+          PACKAGE_URL: github.com/securego/gosec/v2/cmd/gosec
+          PACKAGE_VERSION: ${{ steps.setup-env.outputs.gosec-version }}
+        run: go install ${PACKAGE_URL}@${PACKAGE_VERSION}
+      - name: Install Staticcheck
```

Comment:
Please move this below the shfmt installation so this is alphabetically sorted.

Reply:
@mcdonnnj please see https://github.com/cisagov/skeleton-generic/pull/138/commits/63fb67ec88273985665541aa82509e68ac389578

Reply:
@jasonodoom - You should really list @mcdonnnj as a co-author on commit 63fb67e since it was his idea.